ui/appui-react/src/appui-react/widgets/WidgetConfig.ts

```diff
@@ -0,0 +1,14 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module Widget
+ */
+
+import { WidgetProps } from "./WidgetProps";
+
+/** Configuration from which a widget is created.
+ * @beta
+ */
+export type WidgetConfig = WidgetProps; // eslint-disable-line deprecation/deprecation
```

Comment:
i dont get the purpose of this? is this just an alias/rename we want to do for 4.0 to keep the Config suffix name consistency ?

Reply:
Yes, given that this is a UI package, props are typically "parameters of a react component", which is no longer the case, that's why we changed the name.